contrib/local/02-contracts.sh

```diff
@@ -25,7 +25,7 @@ wasmd tx wasm instantiate "$CODE_ID" "$INIT" --admin=$(wasmd keys show validator
   --from validator --amount="100ustake" --label "local0.1.0" \
   --gas 1000000 -y --chain-id=testing -b block | jq
 
-CONTRACT=$(wasmd query wasm list-contract-by-code "$CODE_ID" -o json | jq -r '.contract_infos[-1].address')
+CONTRACT=$(wasmd query wasm list-contract-by-code "$CODE_ID" -o json | jq -r '.contracts[-1]')
```

Comment:
Shouldn't this be plural?

```suggestion
CONTRACT=$(wasmd query wasm list-contracts-by-code "$CODE_ID" -o json | jq -r '.contracts[-1]')
```

Reply:
yes, and it is always 😕  
I have opened https://github.com/CosmWasm/wasmd/issues/496 already to address this via an alias so that old docs/tutorials still make sense

Reply:
Ah, nice ticket and solution. Then `list-contracts-by-code` can become the primary command without braking.